be/test/storage/rowset/segment_v2/binary_dict_page_test.cpp

```diff
@@ -75,8 +77,18 @@ class BinaryDictPageTest : public testing::Test {
         ASSERT_EQ(slices.size(), dict_page_decoder->count());
 
         // decode
+        Slice encoded_data = s.slice();
+        PageFooterPB footer;
+        footer.set_type(DATA_PAGE);
+        DataPageFooterPB* data_page_footer = footer.mutable_data_page_footer();
+        data_page_footer->set_nullmap_size(0);
+        std::unique_ptr<char[]> page = nullptr;
+
+        Status st = DataDecoder::decode_page(&footer, 0, DICT_ENCODING, &page, encoded_data);
```

Comment:
```suggestion
        Status st = DataDecoder::decode_page(&footer, 0, DICT_ENCODING, &page, decoded_data);
```
the same for following